src/apps/FEMSWEEP.cpp

```diff
@@ -110,7 +110,13 @@ void FEMSWEEP::setUp(VariantID vid, size_t RAJAPERF_UNUSED_ARG(tune_idx))
 
 void FEMSWEEP::updateChecksum(VariantID vid, size_t tune_idx)
 {
+#if defined(RAJA_ENABLE_HIP)
+  // The AMD CPU checksum is off starting at the 10's digit. AMD GPU and NVIDIA GPU results match.
+  // This truncation will be removed when that issue is resolved.
+  checksum[vid][tune_idx] += (std::trunc(calcChecksum(m_Xdat, m_Xlen, checksum_scale_factor , vid) / 100.0) * 100.0);
```

Comment:
Might be good to check different compiler versions to see if this is compiler specific...

Reply:
Can we use the checksum_scale_factor to scale down the checksum so the difference is deemed acceptable?